src/storage/test/GetNeighborsTest.cpp

```diff
@@ -184,7 +208,9 @@ TEST(GetNeighborsTest, PropertyTest) {
 
     ASSERT_EQ(0, (*resp.result_ref()).failed_parts.size());
     // vId, stat, player, - teammate, - serve, expr
-    QueryTestUtils::checkResponse(*resp.vertices_ref(), vertices, over, tags, edges, 1, 6);
+    size_t expectColumnCount = 4;
```

Comment:
update the comment as well, so as `expectColumnCount` you add in this PR

Reply:
ok